README.md

```diff
@@ -25,11 +25,25 @@ lang, _ = GetLanguageByContent("foo.m", "<matlab-code>")
 fmt.Println(lang)
 // result: Matlab
 
-lang, _ = GetLanguageByContent("bar.m", "<pbjective-c-code>")
+lang, _ = GetLanguageByContent("bar.m", "<objective-c-code>")
 fmt.Println(lang)
 // result: Objective-C
+
+// all strategies together
+lang := enry.GetLanguage("foo.cpp", "<cpp-code>")
 ```
 
+Developmemt
+-----------
+*enry* re-uses parts of original [linguist](https://github.com/github/linguist) especially data in `languages.yml` to generate internal data structures. In oreder to update to latest upstream run
+
+    make clean code-generate
```

Comment:
I think it's a valid concern but this can be changed in subsequent PRs.

This one just aims to highlight users about the ability to update/generate something.